.github/workflows/workflow-path-to-live.yml

```diff
@@ -30,13 +30,15 @@ jobs:
     name: output workflow variables
     outputs:
       build_identifier: "main"
-      short_sha: ${{ steps.variables.outputs.short_sha }}
+      version_tag: ${{steps.semver_tag.outputs.tag}}
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # pin@v3
-      - name: extract variables for workflow
-        id: variables
-        run: |
-          echo "short_sha=$(echo ${GITHUB_SHA:0:7})" >> $GITHUB_OUTPUT
+      - name: generate semver tag and release
+        id: semver_tag
+        uses: ministryofjustice/opg-github-actions/.github/actions/semver-tag@v2.3.2
```

Comment:
@jamesrwarren i'm just about to push a newer version that fixes a bug when remotely calling the semver tag action  :)

Reply:
ah ok i'll wait then :)

Reply:
@jamesrwarren [v2.4.0](https://github.com/ministryofjustice/opg-github-actions/releases/tag/v2.4.0) is now released :)